Edit: Getting coins (#55)
FalixNodes-Software/help-center: _posts/falix/general/2021-08-10-getting-coins.md

## Joining Promoted Discord Servers

You can join a promoted server and receive FalixCoins for it! Simply go to the [promotions page](https://client.falixnodes.net/join_for_rewards) to see the promoted servers available along with the expiry date and the number of coins you will receive from joining it.

> Do note that you must log out and login to recieve your coins.

Reviewer: ```suggestion
Note that you must log out and log back in to receive your coins.
```

## Joining Promoted Discord Servers

You can join a promoted server and receive FalixCoins for it! Simply go to the [promotions page](https://client.falixnodes.net/join_for_rewards) to see the promoted servers available along with the expiry date and the number of coins you will receive from joining it.

> To receive your coins for joining servers, Click on "[Get FalixCoins for joined servers](https://client.falixnodes.net/updatejoinedservers)", in the join for resources page.

Reviewer 2: ```suggestion
To receive your FalixCoins from joining promoted servers, Click on "[Get FalixCoins for joined servers](https://client.falixnodes.net/updatejoinedservers)", in the join for resources page.
```

Please also add a hyperlink to the join for resources page.

> To receive your FalixCoins from joining promoted servers, Click on "[Get FalixCoins for joined servers](https://client.falixnodes.net/updatejoinedservers)", in the [join for resources page](https://client.falixnodes.net/join_for_rewards).

Reviewer 2: ```suggestion
To receive your FalixCoins from joining promoted servers, Click on "[Get FalixCoins for joined servers](https://client.falixnodes.net/updatejoinedservers)", in the [join for rewards page](https://client.falixnodes.net/join_for_rewards).
```
